### ISSUE TITLE
Duplicate Landmark/Region names in main interface

### ISSUE THREAD

**Issue by togregg** [edrlab/thorium-reader#1716]
When navigating the Thorium main interface by region, when you get to the section of the interface that represents the My Books, Catalogs and Settings options, it announces bookshelf navigation region. The next section that represents Views, Search, All books and add a book is announced as bookshelf navigation region. 

Tested with:

- Thorium Reader v2.0.0 (Windows and macOS)
- Windows 10 Enterprise (Version 21H2  - OS build 19044.1766)
- JAWS Professional (Version 2022.2206.9 ILM)
- MacOS Big Sur (11.6.7)
- VoiceOver

Steps to demonstrate the issue:

1. Launch your screen reader
2. Launch Thorium Reader
3. Using your screen reader, navigate by regions:

JAWS

- Press R to announce Toolbar region
- Press R again to announce Bookshelf Menu navigation region
- Press R again to announce Bookshelf Menu navigation region
- Press Insert + F3 to bring up the Virtual HTML feature window
- Arrow down or R to get to Regions List and then press Enter
- In the document regions window, Bookshelf menu Navigation appears twice (see image below)
![image](https://user-images.githubusercontent.com/92333167/175571764-43d8c0d1-ebe9-4411-962e-780e5e9af2c4.png)

VoiceOver

- Press Command + Option + U to bring up the rotor
- Press the left or right arrows keys to get to Landmarks
- Press down arrow to announce toolbar region
- Press down arrow again to announce Bookshelf menu navigation
- Press down arrow again to announce Bookshelf menu navigation
- In the Landmarks section of the rotor window, Bookshelf menu navigation appears twice (see image below)
![image](https://user-images.githubusercontent.com/92333167/175572753-54cc2c30-5a53-4eff-b731-82d4ea2677bc.png)

Video demonstrating the issue - [Thorium Reader_Duplicate Landmark Names in Main Interface](https://iu.mediaspace.kaltura.com/media/t/1_fan4s05a)

**Comment by danielweck** [comment]
Thank you very much for reporting this problem so clearly.

**Comment by togregg** [comment]
Verified issue still occurs with version 2.1.0 of Thorium Reader. Any idea where on the roadmap this will be addressed?

**Comment by danielweck** [comment]
Sorry for the delay in fixing this seemingly trivial issue. I tested various combinations of screen reader announcements (I used Mac OS Voice Over) while aiming / hoping not to necessitate adding a new translated label in the user interface. I reused the label that says "menu" for the secondary navigation landmark (the primary continues to say "Bookshelf menu"). The secondary menu is displayed in the local bookshelf as well as when browsing OPDS feeds (second tab named "catalogs")
